docs/features/virtual_folders.rst

```diff
@@ -0,0 +1,96 @@
+.. _virtual_folders:
+
+Virtual Folders
+===============
+
+.. versionadded:: 2.7
+
+Virtual folders provide a way to group translations based on any criteria,
+including a file across all the languages in a project, or files on specific
+locations. Virtual folders have priority, so they can be used to allow
+translators to focus on the most important work.
+
+
+.. _virtual_folders#attributes:
+
+Virtual folders attributes
+-----------------------------------
+
+Virtual folders have several attributes:
+
+- A mandatory lowercase name,
+- A mandatory location,
+- An optional priority,
+- An optional browsability flag,
+- An optional description,
+- An optional filtering rule.
+
+Each virtual folder must have a unique combination of name and location. This
+means that there can exist two different virtual folders with the same name if
+they have different locations.
+
+The priority defaults to ``1`` and accepts any value greater than ``0``,
+including numbers with decimals, like ``0.75``.
+
+By default virtual folders cannot be browsed: they won't be displayed, but
+they are still used for sorting.
+
+Also the virtual folders can have a description which might be useful to tell
+the translators which is the grouping criteria or provide additional
+instructions. This might be handy when using the virtual folders as goals.
+
+The filtering rule specifies which translation units are included within a
+virtual folder. Currently it consists of a list of file paths relative to the
+virtual folder location.
+
+
+.. _virtual_folders#apply:
+
+Adding and updating virtual folders
+-----------------------------------
+
+To add or modify the properties of virtual folders use the
+:ref:`add_vfolders <commands#add-virtual-folders>` management command.
+
+This command imports a JSON file holding a list of virtual folders, and the
+files included on each virtual folder along with all their attributes. Check
+the specs for the :ref:`JSON format <virtual_folders#json-format>` in order to
+know how to craft a JSON file that fits your needs.
+
+
+.. _virtual_folders#translate:
+
+Translating virtual folders
+---------------------------
+
+If a virtual folder applies in the current location, then clicking on the links
+on the overview page will provide the units in priority order when translating
+in the editor. The priority sorting on the translation editor is calculated
+taking into account all the applicable virtual folders in the current location,
+including the not browsable ones.
+
+
+.. _virtual_folders#json-format:
+
+Format for the JSON file
+------------------------
+
+The JSON file used to import virtual folders consists of a list of virtual
+folder definitions with the following fields:
+
+- A mandatory lowercase **name** for the virtual folder,
```

Comment:
@julen It will fail if the location is the same and the names lowercased are the same.